webpackConfig/build-min.js

```diff
@@ -0,0 +1,17 @@
+const UglifyJSPlugin = require('uglifyjs-webpack-plugin');
```

Comment:
let's rename this to `config/webpack.config.prod.js`, it's more consistent with other projects